packages/luna/src/components/Chat/ChatRoom/ChatRoom.tsx

```diff
@@ -68,70 +86,89 @@ export const ChatRoom: React.FC<{
         }
       )}
     >
-      <div className="tw-w-14 tw-h-14 tw-rounded-full tw-overflow-hidden">
-        <Avatar src={image} alt={name} seed={userId.toString()} />
+      <div className="tw-w-14 tw-h-14 tw-relative tw-rounded-full tw-shrink-0">
+        <div className="tw-overflow-hidden tw-rounded-full tw-relative tw-w-full tw-h-full">
+          <Avatar src={image} alt={name} seed={userId.toString()} />
+        </div>
+        {online && !isActive ? (
+          <div className="tw-absolute tw-bottom-0 tw-right-1 tw-z-activityIndicator tw-w-3 tw-h-3 tw-rounded-full tw-bg-brand-500 tw-border-[.5px] tw-border-natural-50 " />
+        ) : null}
       </div>
       <div className="tw-text-right tw-grow">
         <Typography
-          className={cn({
-            "tw-text-natural-50 dark:tw-text-secondary-50": isActive,
-            "tw-text-natural-950 dark:tw-text-natural-50": !isActive,
-          })}
+          className={cn(
+            {
+              "tw-text-natural-50 dark:tw-text-secondary-50": isActive,
+              "tw-text-natural-950 dark:tw-text-natural-50": !isActive,
+            },
+            "tw-flex tw-gap-2 tw-items-center"
+          )}
           weight="bold"
           element="body"
         >
           {name}
+          {isMuted ? (
+            <Muted className={cn(isActive && "[&>*]:tw-stroke-natural-50")} />
+          ) : null}
         </Typography>
         <Typography
           tag="div"
           element="caption"
-          className={cn("tw-line-clamp-1 tw-mt-2 tw-text-right", {
+          className={cn(" tw-mt-2 tw-text-right tw-flex tw-gap-1", {
             "tw-text-natural-50 dark:tw-text-secondary-50": isActive,
-            "tw-text-brand-700 dark:tw-text-brand-400": !isActive,
+            "tw-text-natural-600": !isActive,
           })}
         >
+          {owner && !isActive ? (
+            <div className="tw-w-[21px] tw-h-[21px] tw-flex tw-items-center tw-justify-center">
```

Comment:
I think icons size may change "discretely" with respect to the screen. If the user is rendering the application from a modern wide screen, `21px` would be very hard to see. And If he/she rendering the application from a mini mobile device, it (the 21px) might be too much.
We may add this as a new task in ClickUp (especially, if we need to audit it in more than one spot in the code base).

Reply:
Great job noticing this, but I think this will be added in the responsive, in the next sprint Inshaa' Allah